.github/workflows/deploy-xpr-testnet.yaml

```diff
@@ -69,11 +69,11 @@ jobs:
             build-kubernetes
         env:
           # general
-          NAMESPACE: proton-testnet-dashboard
-          INGRESS_HOST: proton-testnet.antelope.tools
-          INGRESS_GRAPHQL_HOST: graphql-proton-testnet.antelope.tools
+          NAMESPACE: xpr-testnet-dashboard
```

Comment:
keep namespace the same to avoid creating a new instance on the servers: `NAMESPACE: proton-testnet-dashboard`